user-guide/Standard_Apps/Monitoring_Solutions/META_WhatsApp_Messaging/META_WhatsApp_Messaging.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: META_WhatsApp_Messaging
+---
+
+# META WhatsApp Messaging solution
+
+The META WhatsApp Messaging solution acts as a link between the [Alarm Console](xref:Working_with_the_Alarm_Console) (1) and the WhatsApp API, allowing critical alarms to be sent directly to designated users' WhatsApp devices (2).
```

Comment:
```suggestion
The META WhatsApp Messaging solution acts as a link between the [Alarm Console](xref:Working_with_the_Alarm_Console) (1) and the WhatsApp API, allowing critical alarms to be sent directly to designated users' devices with WhatsApp installed (2).
```

I don't think "WhatsApp devices" is accurate. How about rephrasing it to "devices with WhatsApp installed"?